cmd/genji/shell/shell.go

```diff
@@ -72,6 +78,55 @@ func stdinFromTerminal() bool {
 	return true // data is from terminal
 }
 
+func buildInsertQuery(table string, data string) string {
+	var b strings.Builder
+
+	b.WriteString("INSERT INTO ")
+	b.WriteString(fmt.Sprintf("%s", table))
+	b.WriteString(" VALUES ")
+	if data == "" {
+		return b.String()
+	}
+
+	if !strings.HasPrefix(data, "{") {
+		b.WriteString("{\"\":")
+		b.WriteString(data)
+		b.WriteString("}")
+	} else {
+		b.WriteString(data)
+	}
+
+	return b.String()
+}
+
+// RunInsertQuery execute Insert query from command line.
+func RunInsertQuery(opts *Options) error {
+	var sh Shell
+	sh.opts = opts
+	for _, d := range opts.Data {
+		err := sh.runQuery(buildInsertQuery(opts.Table, d))
+		if err != nil {
+			return err
+		}
+	}
+
+	// block for piped input
+	ran, err := sh.runPipedInput()
+	if err != nil {
+		return err
+	}
+
+	if ran {
+		return nil
+	}
+
+	if len(opts.Data) == 0 {
+		return fmt.Errorf("error: insert -f %q -t %q %q", sh.opts.DBPath, sh.opts.Table, sh.opts.Data)
```

Comment:
If you want to display the usage, don't do it by hand, use the framework instead